packages/commonjs/test/snapshots/function.js.md

```diff
@@ -241,6 +241,36 @@ Generated by [AVA](https://avajs.dev).
       `,
     }
 
+## class-body-property-keys-are-special-keywords
+
+> Snapshot 1
+
+    {
+      'main.js': `'use strict';␊
+      ␊
+      Object.defineProperty(exports, '__esModule', { value: true });␊
+      ␊
+      var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};␊
+      ␊
+      function commonjsRequire(path) {␊
+      	throw new Error('Could not dynamically require "' + path + '". Please configure the dynamicRequireTargets or/and ignoreDynamicRequires option of @rollup/plugin-commonjs appropriately for this require call to work.');␊
+      }␊
+      ␊
+      var main = {};␊
+      ␊
+      class Rollup {␊
+        define = undefined;␊
+        require = commonjsRequire;␊
+        global = commonjsGlobal;␊
```

Comment:
I am not sure this is correct. We were defining a class with properties define, require and global, but as they do not have an initializer, this is not like object shorthand properties but rather they are initialized to undefined, see also https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields#description

I assume this will just make the code simpler, though.

Reply:
After looking at this again, I find out that the function `commonjsRequire` is not useful. So I agree with you that the `require` property should not be transformed into `require = commonjsRequire`. The `define` property is the same as well. As for `global`, I think it could be transformed into `global = commonjsGlobal` if the option `ignoreGlobal` is not `true`.

Reply:
Oh, I get it now. Thanks for your patient explanation!